tests/ert_tests/shared/test_log_process_usage.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from ert_shared.main import log_process_usage
+
+
+def test_valid(caplog):
+    caplog.set_level(logging.INFO)
+    log_process_usage()
+
+    assert len(caplog.records) == 1
+
+
+def test_unsupported_platform(caplog, monkeypatch):
+    """
+    This test supposes that some exception occurs due to the current system not
+    supporting the 'resource' package. The correct behaviour is to silently
```

Comment:
Which systems does not support this?

Reply:
Windows, for example and there are differences between what the different POSIX OSes provide.

But the main purpose of this test is for the logging to fail gracefully (that is, not throw an exception or terminate the process but just log).